<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="AnimationViewer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string>Media</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>35</y>
       <width>361</width>
       <height>381</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="PlaylistManager">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>361</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string>Playlist</string>
    </property>
    <widget class="QListWidget" name="playlist">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>221</width>
       <height>391</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="downbtn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>160</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addbtn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removebtn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
    <widget class="QPushButton" name="upbtn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="MediaControl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>781</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Control Box</string>
    </property>
    <widget class="QPushButton" name="startbtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pausebtn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resumebtn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Resume</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopbtn">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </widget>
   <zorder>PlaylistManager</zorder>
   <zorder>MediaControl</zorder>
   <zorder>AnimationViewer</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
